runtime-linux.md

```diff
@@ -1,8 +1,18 @@
 # Linux Runtime
 
 ## File descriptors
+
 By default, only the `stdin`, `stdout` and `stderr` file descriptors are kept open for the application by the runtime.
+The runtime MAY pass additional file descriptors to the application to support features such as [socket activation](http://0pointer.de/blog/projects/socket-activated-containers.html).
+Some of the file descriptors MAY be redirected to `/dev/null` even though they are open.
+
+## Dev symbolic links
 
-The runtime may pass additional file descriptors to the application to support features such as [socket activation](http://0pointer.de/blog/projects/socket-activated-containers.html).
+After the container has `/proc` mounted, the following standard symlinks MUST be setup within `/dev/` for the io.
```

Comment:
I'd recommend wording like the existing [“MUST be made available”][1] or [“MUST … supply”][2] instead of “MUST be setup”.  Config authors and runtime callers should only care about the links being there, and shouldn't care about whether it was the runtime that set them up.  And runtimes that bind-mount `/dev` (or similar) into the container may already have these links and not need to create them themselves.  I'd also like to make the timing for these required objects explicit (see the third bullet [here][3]).  And cross-links between these “paths the runtime must setup if missing” sections would be good too.

[1]: https://github.com/opencontainers/runtime-spec/blob/v0.5.0/config-linux.md#default-file-systems
[2]: https://github.com/opencontainers/runtime-spec/blob/v0.5.0/config-linux.md#default-devices
[3]: https://github.com/opencontainers/runtime-spec/pull/164#issuecomment-138984824